modules/howtos/examples/AsyncOperations.java

```diff
@@ -112,7 +113,8 @@ public static void main(String... args) {
       List<String> docsToFetch = Arrays.asList("airline_1316", "airline_13391", "airline_1355");
 
       List<GetResult> results = Flux.fromIterable(docsToFetch)
-          .flatMap(key -> reactiveCollection.get(key).retryBackoff(10, Duration.ofMillis(10))).collectList().block();
+          .flatMap(key -> reactiveCollection.get(key).retryWhen(Retry.backoff(10, Duration.ofMillis(10)))).collectList()
+          .block();
```

Comment:
Is this related to the retryWhen change, or something else?
(Not important, just curious)

Reply:
aye, `retryBackoff` was deprecated and replaced with `retryWhen`.

Reply:
sorry, I meant the `.block()` line specifically, which didn't seem *obviously* part of that API change, so just curious :)

Reply:
oh, `.block()` was there before, I haven't changed that

Reply:
I think it's just formatting probably that might have confused things.

Reply:
ah! yes! moar coffee!